demo/scripts/controls/ContentModelEditorMainPane.tsx

```diff
@@ -110,6 +111,7 @@ class ContentModelEditorMainPane extends MainPaneBase<ContentModelMainPaneState>
     private formatPainterPlugin: ContentModelFormatPainterPlugin;
     private pastePlugin: ContentModelPastePlugin;
     private sampleEntityPlugin: SampleEntityPlugin;
+    private tableEditPlugin: TableEditPlugin;
```

Comment:
Can you also do the change in demo\scripts\controls\StandaloneEditorMainPane.tsx?

You can test it with parameter "?cm=2"

Later we will use this new page to replace the old one.